### PR TITLE
Random Battle: Change joke Pokemon

### PR DESCRIPTION
**Pull request by KrisXV**
discussed in dev, comment on anything i should do differently

### CODE REVIEW

File: data/random-teams.js [hunk 1]
```diff
@@ -1392,9 +1392,9 @@ class RandomTeams extends Dex.ModdedDex {
 		} else if (hasMove['magikarpsrevenge']) {
 			// PoTD Magikarp
 			item = 'Choice Band';
-		} else if (template.species === 'Rotom-Fan') {
+		} else if (template.species === 'Stunfisk' && !teamDetails['sand']) {
 			// This is just to amuse Zarel
```
Comment (Zarel) [inline data/random-teams.js]: Wow preserving the comment... okay fine.

I would probably set `ability = 'Limber'` here instead of up there.

**Comment by TheImmortal** [comment]
What’s the reason for the Sand check?

**Comment by KrisXV** [comment]
There was discussion in dev about having Stunfisk only hold Cheri Berry if its ability was Limber, but in hindsight this wasn't the best approach to it

**Comment by TheImmortal** [comment]
Obviously I’d like that better instead of the 100% meme chance. @Zarel?

File: data/random-teams.js [hunk 2]
```diff
@@ -1278,8 +1278,6 @@ class RandomTeams extends Dex.ModdedDex {
 					rejectAbility = !teamDetails['hail'];
 				} else if (ability === 'Lightning Rod') {
 					rejectAbility = template.types.includes('Ground');
-				} else if (ability === 'Limber') {
-					rejectAbility = template.types.includes('Electric');
```
Comment (Zarel) [inline data/random-teams.js]: This should probably not be deleted.

**Comment by Zarel** [comment]
Its ability should never be Limber though?

**Comment by Zarel** [comment]
We can rotate it out at some point I guess.

**Comment by Zarel** [comment]
Also we _should_ give it a few more levels in exchange for the memeing.

**Comment by Zarel** [comment]
I did literally the opposite of approve this change, @TheImmortal. I requested changes and wasn't even given the chance of waiting for those changes to actually happen!

File: data/random-teams.js [hunk 3]
```diff
@@ -1279,7 +1279,7 @@ class RandomTeams extends Dex.ModdedDex {
 				} else if (ability === 'Lightning Rod') {
 					rejectAbility = template.types.includes('Ground');
 				} else if (ability === 'Limber') {
-					rejectAbility = template.types.includes('Electric');
+					// rejectAbility = template.types.includes('Electric');
```
Comment (Zarel) [inline data/random-teams.js]: Like I specifically said not to comment out this code!

**Comment by TheImmortal** [comment]
You made it 100%. I was making it dependent on what ability it rolls. It can roll Limber, which is why that line was removed.

**Pull request merged**